nav2_costmap_2d/plugins/inflation_layer.cpp

```diff
@@ -78,7 +78,12 @@ InflationLayer::InflationLayer()
 
 InflationLayer::~InflationLayer()
 {
+  auto node = node_.lock();
+  if (dyn_params_handler_ && node) {
+    node->remove_on_set_parameters_callback(dyn_params_handler_.get());
+  }
   dyn_params_handler_.reset();
+  node.reset();
```

Comment:
I don't think this is necessary? (or all)

Reply:
I've removed them now.